contrib/babelfishpg_common/sql/upgrades/spatial_types--3.3.0--3.4.0.sql

```diff
@@ -1281,20 +1336,87 @@ CREATE OR REPLACE FUNCTION sys.STDistance_helper(geog1 sys.GEOGRAPHY, geog2 sys.
 
 CREATE OR REPLACE FUNCTION sys.GEOGRAPHY_helper(bytea)
 	RETURNS sys.GEOGRAPHY
-	AS '$libdir/postgis-3','geography_from_binary'
+	AS '$libdir/postgis-3','LWGEOM_from_bytea'
 	LANGUAGE 'c' IMMUTABLE STRICT PARALLEL SAFE;
 
-CREATE OR REPLACE FUNCTION sys.bpcharToGeography_helper(sys.bpchar)
+DO $$
+DECLARE
+    exception_message text;
+BEGIN
+    -- Rename bpcharToGeography_helper function for dependencies
+    ALTER FUNCTION sys.bpcharToGeography_helper(sys.bpchar) RENAME TO bpcharToGeography_helper_deprecated_3_4_0;
+
+    -- === DROP bpcharToGeography_helper_deprecated_3_4_0
+    CALL sys.babelfish_drop_deprecated_object('function', 'sys', 'bpcharToGeography_helper_deprecated_3_4_0');
```

Comment:
This is the first time we are introducing `bpcharToGeography_helper` function so why are dropping this? Just update the existing occurrence directly.

Reply:
It's not the first time we are introducing ```bpcharToGeography_helper``` function, we had it earlier as well, reference: https://github.com/babelfish-for-postgresql/babelfish_extensions/blob/b555f14ea094e67d26bed4938c51c7429163e769/contrib/babelfishpg_common/sql/upgrades/spatial_types--3.2.0--3.3.0.sql#L914
Now we have added a new parameter in this function so dropping the last version of this function and then creating a new version

Reply:
Nope, this is the first version we are introducing the geospatial feature and all it's components so no functions should exists earlier. The example you gave above should not apply since we disabled the feature for the earlier versions.

Reply:
Got it and reverted it

Reply:
How did we disable the feature earlier? By reverting the code change or using a GUC to give error? Making sure that there was not entry in upgrade scripts for new functions which will be created even if feature is not available via GUC or similar.

Reply:
We had a compile time flag setted in the Github actions, so we disabled that flag and all the upgrade script code will only be present when the flag is enabled, so this is the first time we are enabling the flag and so their were no prior entry in upgrade scripts for new functions.
Reference:
https://github.com/babelfish-for-postgresql/babelfish_extensions/blob/b555f14ea094e67d26bed4938c51c7429163e769/contrib/babelfishpg_common/sql/upgrades/babelfishpg_upgrades.in#L11